OrbitGl/TimeGraph.cpp

```diff
@@ -802,6 +898,43 @@ std::shared_ptr<GpuTrack> TimeGraph::GetOrCreateGpuTrack(
   return track;
 }
 
+//-----------------------------------------------------------------------------
+static void SetTrackNameFromRemoteMemory(std::shared_ptr<Track> track,
+                                         uint64_t string_address,
+                                         OrbitApp* app) {
+  app->GetThreadPool()->Schedule([track, string_address, app]() {
+    int32_t pid = Capture::GTargetProcess->GetID();
+    const auto& process_manager = app->GetProcessManager();
+    auto error_or_string =
+        process_manager->LoadNullTerminatedString(pid, string_address);
+
+    if (error_or_string.has_value()) {
+      app->GetMainThreadExecutor()->Schedule([track, error_or_string]() {
+        track->SetName(error_or_string.value());
+        track->SetLabel(error_or_string.value());
+      });
+    } else {
+      ERROR("Setting track name from remote memory: %s",
+            error_or_string.error().message());
+    }
+  });
+}
+
+//-----------------------------------------------------------------------------
```

Comment:
these commends have been removed from the code recently. Can you remove them from this change as well?

Reply:
Has this been addressed?